GitHubDashboard/ClientApp/app/components/query/count.component.ts

```diff
@@ -51,5 +55,16 @@ class QueryCountResult {
                 this.query += `label%3A${label} `;
             }
         }
+
+        if (excludedMilestone)
```

Comment:
Nit: Style is brace on same line.

Reply:
Fixed

---
In reply to: [247577679](https://github.com/Pilchie/GitHubDashboard/pull/13#discussion_r247577679) [](ancestors = 247577679)